src/community/who-uses-dart.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Who uses Dart
+description: Mobile apps and Google web apps are two of the most common uses for the Dart language.
+toc: false
+---
+
+Many Dart developers write mobile apps for Android, iOS, or both,
+using the [Flutter toolkit.][Flutter]
+To find a list of Flutter apps, visit the [Flutter showcase.][showcase]
+
+[Flutter]: {{site.flutter}}
+[showcase]: {{site.flutter}}/showcase
+
+Google engineers use Dart to create many apps,
+including some that are essential to Google's business.
+For example, if you use the [Google Ads][] web or mobile app,
```

Comment:
Maybe also mention Assistant on Smart Displays with a link to https://developers.googleblog.com/2019/05/Flutter-io19.html ?

Reply:
How about adding this sentence:

Also, the Assistant team at Google uses Dart for features in Smart Displays, as described in [this announcement.](https://developers.googleblog.com/2019/05/Flutter-io19.html)

Reply:
👍